internal/handler/queue.go

```diff
@@ -20,11 +20,16 @@ const queueId key = 0
 
 func GetQueue(w http.ResponseWriter, r *http.Request) {
 	id := r.Context().Value(queueId).(string)
+	uid := r.Context().Value("uid").(string)
```

Comment:
See if you are able to move "uid" into a string constant that you can export from the authentication module. Then can you replace all occurrences of the raw string with the constant?

Reply:
Made the change